packages/button/src/Button.stories.mdx

```diff
@@ -0,0 +1,92 @@
+import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs';
```

Comment:
Copyright

Reply:
mdxjs har elendig støtte for kommentarer fant jeg ut nå. Foreslår at vi dropper copyright i disse filene, eventuelt i resten av eksemplene også.